build/common.sh

```diff
@@ -2,24 +2,63 @@
 
 set -e
 path=$(dirname "$0")
-base=$(cd $path/.. && pwd)
-drush="$PWD/vendor/bin/drush $drush_flags -y -r $base/www"
+pushd $path/..
+base=$(pwd);
 
-if [[ -f .env ]]; then
-  source .env
-else
-  echo "No env file found. Please create one. You can use env.dist as an example."
-  exit 1
-fi
-# Confirm our working directory
-if [ ! -d $base/www ]; then
-  mkdir $base/www
+drupal_base="$base/www"
+
+while getopts ":r:d:" opt; do
+  case $opt in
+    r)
+      # Drupal Root Directory
+      drupal_base="$base/$OPTARG";
```

Comment:
For consistency we can add all double quotes to all variables. Even to this line
```
base="$(dirname "$0")/.."
```